### ISSUE TITLE
2.3.0-RC5 - RSS feed broken

### ISSUE THREAD

**Issue by brianjking** [CachetHQ/Cachet#1832]
Before submitting your issue, please make sure that you've checked all of the checkboxes below.

- [x] You're running the [latest release](https://github.com/CachetHQ/Cachet/releases/latest) version of Cachet.
- [x] Ensure that you're running at least PHP 5.5.9, you can check this by running `php -v`
- [x] You've ran `rm -rf bootstrap/cache/*` from the root of your Cachet installation.

To help us better understand your issue, please answer the following — cheers!

### Your setup

- *What version of Cachet?* 2.3.0 - RC1
- *What database driver? MySQL? Postgres? SQLite?* MySQL
- *What version of PHP?* 5.6.16

### Expected behaviour

* RSS feed to display 

### Actual behaviour

```
This page contains the following errors:

error on line 19 at column 46: xmlParseEntityRef: no name
Below is a rendering of the page up to the first error.
```

### Steps to reproduce

*If your issue requires any specific steps to reproduce, please outline them here.*

**Comment by jbrooksuk** [comment]
This might be something with your content. Can I see the status page please? :)

**Comment by jbrooksuk** [comment]
Ping @brianjking, I can't replicate this issue at all so unfortunately I need a little more information to go on :)

**Comment by brianjking** [comment]
@jbrooksuk http://monitor.duelingdogs.net

**Comment by brianjking** [comment]
@jbrooksuk Another thing I noticed is each individual component used to be able to be subscribed to, however, this is now missing. However, subscription is still available for the whole site.

Was this removed in 2.3.0-RC1?

Thanks!

**Comment by brianjking** [comment]
@jbrooksuk 

I upgraded my 2nd site to 2.3.0-RC3 and the RSS function is broken there too.

http://monitor.dockdogs.com/rss

```
This page contains the following errors:

error on line 88 at column 37: xmlParseEntityRef: no name
Below is a rendering of the page up to the first error.
```

**Comment by brianjking** [comment]
Upgraded http://monitor.duelingdogs.net to `2.3.0-RC4` and then `2.3.0-RC5` both continue the RSS feed errors and lack of subscription by individual components.

Thanks, please let me know how I can help you get to the bottom of this.

**Comment by jbrooksuk** [comment]
I _think_ I know what issue is.

Regarding the subscriptions, pinging @joecohens - a lot of people have mentioned the individual subscription thing. Anything we can do?

**Comment by brianjking** [comment]
Okay, thanks. Let me know how I can help you track it down.

The subscription issue is less of a problem for me since it appears the overall component subscription still works. However, if I can be of any assistance to @jbrooksuk & @joecohens let me know.

Thanks!

**Comment by jbrooksuk** [comment]
I've found the issue. It lies within the package we use to generate the feed. https://github.com/RoumenDamianoff/laravel-feed/pull/53

**Comment by jbrooksuk** [comment]
@CachetHQ/owners we need to close this **only** once the package has been released.

**Comment by GrahamCampbell** [comment]
Meh, we can just pull in a dev version. I'll do it now.

**Comment by brianjking** [comment]
@jbrooksuk @GrahamCampbell Just installed 2.3.0-RC6 on both sites and the RSS feeds seem to validate and work properly now.

Subscribe to individual components still appears to be missing, but global subscribe still seems to be working.

Thanks!

**Comment by jbrooksuk** [comment]
> Subscribe to individual components still appears to be missing, but global subscribe still seems to be working.

We might need to change this a little, but you click the **Subscribe** button, and then you can pick your components.

**Comment by GrahamCampbell** [comment]
> Just installed 2.3.0-RC6 on both sites and the RSS feeds seem to validate and work properly now.

RSS feeds should be fixed on both 2.2.4 and 2.3.0-RC6. :)
